pkg/convert/pot.go

```diff
@@ -0,0 +1,70 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+func ToPot(m model.Messages) ([]byte, error) {
+	var b bytes.Buffer
+
+	b.WriteString(fmt.Sprintf("\"Language: %s\n", m.Language))
+
+	for _, message := range m.Messages {
+		if message.Description != "" {
+			b.WriteString(fmt.Sprintf("#. %s\n", message.Description))
+		}
+
+		if message.Fuzzy {
+			b.WriteString("#, fuzzy\n")
+		}
+
+		b.WriteString(fmt.Sprintf("msgid \"%s\"\n", message.ID))
+
+		b.WriteString(fmt.Sprintf("msgstr \"%s\"\n", message.Message))
```

Comment:
- What happens if `message.Message` value is `Hello "World"`? Currently, it writes `msgstr "Hello "World""` to POT. How to escape double quotes in POT?
- What happens if `message.Message` contains a multiline value?